bin/src/_main.rs

```diff
@@ -27,17 +29,22 @@ use middleware::meta_rules::{MetaRules, MetaRulesConfig};
 use middleware::Executor;
 
 use pin_utils::pin_mut;
+use rand::Rng;
 use state::{AgentState, FileId, SpanVec};
 use std::collections::HashMap;
 use std::sync::Arc;
 use tokio::signal::*;
 use tokio::sync::Mutex;
+use tokio::time::sleep;
 use tokio::time::Duration;
 
 #[cfg(feature = "dep_audit")]
 use crate::dep_audit;
 use crate::stream_adapter::{StrictOrLazyLineBuilder, StrictOrLazyLines};
 
+pub static REPORTER_LEASE_NAME: &str = "logdna-agent-reporter-lease";

```

Comment:
Const?

Reply:
fixed and moved down a bit - its by other vars though, let me know if you have a better spot.